parcels/particleset.py

```diff
@@ -420,7 +420,8 @@ def execute(self, pyfunc=AdvectionRK4, endtime=None, runtime=None, dt=1.,
         :param movie_background_field: field plotted as background in the movie if moviedt is set.
                                        'vector' shows the velocity as a vector field.
         :param verbose_progress: Boolean for providing a progress bar for the kernel execution loop.
-        :param postIterationFunctions: Array of functions that are to be called after each iteration (post-process, non-Kernel)
+        :param postIterationCallbacks: Array of functions that are to be called after each iteration (post-process, non-Kernel)
```

Comment:
Add in the docstring that these two functions are most useful for optimisation purposes?

Reply:
if you think we should add it there, we can. The function is flexible, as we use if for performance measurements as well as garbage collection - which I assume scientists with large-scale case studies or long-running simulations may want to make use of.

Reply:
Well, I'd like to make sure that typical users don't expect this to be something they _have_ to use. So even the word '(Optional)' in front would help?

Reply:
k, let's do that